packages/jest-cli/src/__tests__/SearchSource-test.js

```diff
@@ -50,10 +51,23 @@ describe('SearchSource', () => {
       });
     });
 
-    it('supports ../ paths and unix separators', () => {
+    // micromatch doesn't support '..' through the globstar ('**') to avoid
+    // infinite recursion.
+
+    it('supports ../ paths and unix separators via textRegex', () => {
```

Comment:
Good catch, this used to rely on the default, so I've changed it to explicitly set a regex.